pages/events/index.vue

```diff
@@ -83,9 +83,38 @@
           </div>
         </template>
         <template slot="extra-info">
-          <div class="event-page__start-an-event">
+          <div class="event-page__section">
+            <h3>Follow our event calendar</h3>
+            <p class="event-page__section__description">
+              <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
+              Stay up to date with all of our scheduled events by following our calendar. You can view the calendar by visiting <AppLink v-bind="qiskitCalendarLink">{{ qiskitCalendarLink.label }}</AppLink>, or subscribe to it by adding to the calendar app of your choice.
+            </p>
+            <div class="event-page__tabs">
+              <cv-tabs aria-label="Calendar applications">
+                <cv-tab
+                  v-for="(calendar, calendarIndex) in calendars"
+                  :id="`tab-${calendar.name}`"
+                  :key="calendarIndex"
```

Comment:
To use index as key is a bad practice, you can read the reason [here](https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop#:~:text=Because%20arrays%20are,using%20an%20index.). Although in this case we will not have that issue because we are not mutating the array, I think we should use good practice and use the `name` instead of the `index` as `key`

Reply:
And if you use the `name` as `key`, you can do this:

```ts
v-for="{ name } in calendars"
```

Reply:
I did a few things to update this:

- renamed `calendars` to `calendarsInstructions`
- inside that array, instead of calling it a `component` property, I'm now calling it `instructions`
  - reason: so that on [this line](https://github.com/Qiskit/qiskit.org/pull/2955/files#diff-e89e9da7098c8b828fffda21f9b905c5668d8726f92332ddd3612c55d7bc0d9eR110), I can avoid writing `<component :is="component">`

- updated to `v-for="{ name, instructions } in calendars"`


please let me know if this makes sense

Reply:
makes totally sense, thank you!